cmd/span.go

```diff
@@ -74,6 +75,10 @@ func startSpan() (context.Context, trace.Span, func()) {
 	ctx, span := tracer.Start(ctx, spanName, startOpts...)
 	span.SetAttributes(cliAttrsToOtel(spanAttrs)...) // applies CLI attributes to the span
 
+	if spanStatus == true {
+		span.SetStatus(codes.Error, "error")	
+	}
```

Comment:
I don't remember what the possible values are here but maybe we want a way to be able to pass in .e.g. http 403 v.s. 500?